presto-main/src/main/java/com/facebook/presto/execution/scheduler/DynamicSplitPlacementPolicy.java

```diff
@@ -51,6 +51,6 @@ public void lockDownNodes()
     @Override
     public List<InternalNode> allNodes()
     {
-        return nodeSelector.allNodes();
+        return nodeSelector.activeNodes();
```

Comment:
Is this an intentional change?

Reply:
yes it is. We now have list of activeNodes and allNodes (including dead ones) . hence the change.

Reply:
Yeah, this looks confusing. we should fix the caller to leverage getActiveNodes.

Reply:
Let's change the method name if possible?